### ISSUE TITLE
FileProvider conflicts with other SDKs

### ISSUE THREAD

**Issue by phips28** [kinecosystem/kin-ecosystem-android-sdk#175]
I am having issues with the current FileProvider implementation in your SDK when another SDK uses FileProvider too:

```
:app:processDebugManifest

> Manifest merger failed with multiple errors, see logs

......./android/app/src/main/AndroidManifest.xml:15:13-60 Error:
	Attribute provider#android.support.v4.content.FileProvider@authorities value=(com.troutloud.dvel.provider) from [:react-native-image-crop-picker] AndroidManifest.xml:15:13-60
	is also present at [com.github.kinfoundation.kin-ecosystem-android-sdk:recovery:0.2.0] AndroidManifest.xml:14:13-61 value=(com.kin.ecosystem.recovery).
	Suggestion: add 'tools:replace="android:authorities"' to <provider> element at AndroidManifest.xml:13:9-21:20 to override.
......./android/app/src/main/AndroidManifest.xml:20:17-55 Error:
	Attribute meta-data#android.support.FILE_PROVIDER_PATHS@resource value=(@xml/provider_paths) from [:react-native-image-crop-picker] AndroidManifest.xml:20:17-55
	is also present at [com.github.kinfoundation.kin-ecosystem-android-sdk:recovery:0.2.0] AndroidManifest.xml:19:17-60 value=(@xml/file_provider_paths).
	Suggestion: add 'tools:replace="android:resource"' to <meta-data> element at AndroidManifest.xml to override.
```

I created a fix, PR is on its way.

**Comment by phips28** [comment]
PR Fix: #176

**Comment by phips28** [comment]
Maybe you should also consider renaming `xml/file_provider_paths` to something more unique `xml/kin_file_provider_paths` - in case some other SDK uses the same name? But I dont know if its necessary.

**Comment by OrenZak** [comment]
Thanks, going to be solved on #177.
